egs2/TEMPLATE/cls1/cls.sh

```diff
@@ -444,12 +444,6 @@ if ! "${skip_eval}"; then
         log "Generate '${cls_exp}/run.sh'. You can resume the process from stage 6 using this script"
         mkdir -p "${cls_exp}"; echo "${run_args} --stage 6 \"\$@\"; exit \$?" > "${cls_exp}/run.sh"; chmod +x "${cls_exp}/run.sh"
         _opts=
-        if [ "${max_wav_duration}" ]; then
```

Comment:
This is generally useful to avoid processing very long audio.
If this is for the test sets, it is okay.
If they are applied for the training data, I'm not sure why it was removed.
Can you explain?

Reply:
Current implementation optionally applies length filtering to training data in stage 3 (Line 237).
And does not apply any length filtering while running inference on test sets (Line 454).

These changes were made to handle the earlier suggestion here - https://github.com/espnet/espnet/pull/5998#discussion_r1957416356

We have an option to remove long audio during training, but we never filter the test set.